Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM rust:1.78.0-slim-bullseye as builder
 
+ARG is_ci=false
```

Comment:
Maybe rename this flag to `is_release` or `release`? I think its clearer if the `Dockerfile` exposes the configuration as-is and the CI say "I will build with release"?